{{cookiecutter.project_slug}}/config/settings/production.py

```diff
@@ -99,11 +99,10 @@
 # ------------------------
 {% if cookiecutter.use_whitenoise == 'y' -%}
 STATICFILES_STORAGE = "whitenoise.storage.CompressedManifestStaticFilesStorage"
-{%- endif -%}
-{%- if cookiecutter.cloud_provider == 'AWS' %}
+{%- elif cookiecutter.cloud_provider == 'AWS' and cookiecutter.use_whitenoise == 'n' %}
```

Comment:
As you're using `elif`, there is no need for the second part, it's already known that `use_whitenoise == 'n'`
```suggestion
{%- elif cookiecutter.cloud_provider == 'AWS' %}
```